src/main/java/core/basesyntax/Application.java

```diff
@@ -2,8 +2,8 @@
 
 public class Application {
     public static void main(String[] args) {
+        Lottery uniqueBallForLottery = new Lottery();
```

Comment:
read comments more attentively, I recommended this the last time, uniqueBallForLottery -> lottery